runtime/src/lib.rs

```diff
@@ -200,6 +200,10 @@ pub const OTP: Balance = 1_000_000_000_000;
 pub const MILLIOTP: Balance = 1_000_000_000;
 pub const MICROOTP: Balance = 1_000_000;
 
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	items as Balance * 15 * OTP + (bytes as Balance) * 6 * OTP
```

Comment:
How these values are determined?

Reply:
So we pass to the function: number of items and number of bytes of this items. Items and bytes are multiplied by some default value that we can change in future.